pyatv/interface.py

```diff
@@ -1174,6 +1193,14 @@ async def set_volume(self, level: float) -> None:
         """
         raise exceptions.NotSupportedError()
 
+    @feature(66, "SetDeviceVolume", "Set device volume level.")
```

Comment:
I suggest that we don't add a new method for rather but rather re-use `set_volume` and just add an optional output device as argument instead:

```python
    async def set_volume(self, level: float, output_device: Optional[OutputDevice] = None) -> None:
```